docker/graylog4.x/docker-compose.yml

```diff
@@ -32,7 +32,7 @@ services:
     restart: always
 
   graylog:   # Graylog: https://hub.docker.com/r/graylog/graylog/
-    image: graylog/graylog:4.3.13
+    image: graylog/graylog:5.0.7
```

Comment:
If you look at the folder name, you'll see this is only for the 4.x versions of Graylog.